Source/Blazorise.Bootstrap/Blazorise.Bootstrap.csproj

```diff
@@ -4,6 +4,7 @@
 
   <PropertyGroup>
     <PackageTags>blazorise blazor components bootstrap</PackageTags>
+    <TargetFrameworks>net5.0</TargetFrameworks>
```

Comment:
Can you please remove this from `.csproj` files? The framework is defined only through the shared `.props` file.